internal/tiltfile/tiltfile_state.go

```diff
@@ -1177,3 +1181,24 @@ func (s *tiltfileState) setTeam(thread *starlark.Thread, fn *starlark.Builtin, a
 
 	return starlark.None, nil
 }
+
+func (s *tiltfileState) translateLocal() ([]model.Manifest, error) {
+	var result []model.Manifest
+
+	for _, r := range s.localResources {
+		mn := model.ManifestName(r.name)
+		tm, err := starlarkTriggerModeToModel(s.triggerModeForResource(r.triggerMode))
+		if err != nil {
+			return nil, err
+		}
+		lt := model.NewLocalTarget(r.cmd, r.deps)
+		m := model.Manifest{
```

Comment:
should populate repos/dockerignores [the same way we do for image targets](https://github.com/windmilleng/tilt/blob/009ef1c491001eeb9cb329d97db25378cdbaa251/internal/tiltfile/tiltfile_state.go#L1023) -- the `deps` are the things we feed into the filewatcher, so if one of the deps is e.g. a git repo, we want to ignore the `.git` directory.

Reply:
(and check in a test that they got populated correctly)

Reply:
Hmm, I don't think it makes sense to popular dockerignores. This local resource could be anything, and I might want it to be able to operate on files that I _don't_ want to explicitly include in my build contexts.